### ISSUE TITLE
[Feature] DeepGemm precompile support for DeepSeek V3.2 DSA Indexer

### ISSUE THREAD

**Issue by gary-wjc** [sgl-project/sglang#19313]
### Checklist

- [x] If this is not a feature request but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Motivation

During performance test of disaggregated DeepSeek-V3.2 service using v0.5.9 docker image, I observed that in the token-generation stage there are occasionally lengthy DeepGemm compilation events on the decoder side, although the log showed that DeepGemm warmup have completed early on startup. As I didn't select fast warm-up mode of DeepGemm, I think there might be some code issues here.

I found that several operations in DSA indexer **directly calls deep_gemm method**, where precompilation optimization from [deep_gemm_wrapper](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/layers/deep_gemm_wrapper/entrypoint.py) would not work:

https://github.com/sgl-project/sglang/blob/76d5410e01205af50587c67b709048e23ef319cc/python/sglang/srt/layers/attention/nsa/nsa_indexer.py#L441

https://github.com/sgl-project/sglang/blob/76d5410e01205af50587c67b709048e23ef319cc/python/sglang/srt/layers/attention/nsa/nsa_indexer.py#L567

https://github.com/sgl-project/sglang/blob/76d5410e01205af50587c67b709048e23ef319cc/python/sglang/srt/layers/attention/nsa/nsa_indexer.py#L617

https://github.com/sgl-project/sglang/blob/76d5410e01205af50587c67b709048e23ef319cc/python/sglang/srt/layers/attention/nsa/nsa_indexer.py#L781

https://github.com/sgl-project/sglang/blob/76d5410e01205af50587c67b709048e23ef319cc/python/sglang/srt/layers/attention/nsa/nsa_indexer.py#L827

@fzyzcjy Is it possible to implement fp8_*mqa_logits methods in deep_gemm_wrapper?
